xtp/src/libxtp/tools/gpu_benchmark.cc

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Standard includes
+#include <chrono>
+#include <iomanip>
+#include <numeric>
+#include <string>
+
+#include "votca/tools/property.h"
+#include "votca/xtp/bse_operator.h"
+#include "votca/xtp/eigen.h"
+#include "votca/xtp/logger.h"
+#include "votca/xtp/openmp_cuda.h"
+#include "votca/xtp/orbitals.h"
+#include "votca/xtp/rpa.h"
+#include "votca/xtp/threecenter.h"
+
+// Local private VOTCA includes
+#include "gpu_benchmark.h"
+
+namespace votca {
+namespace xtp {
+
+void GPUBenchmark::ParseOptions(const tools::Property& options) {
+
+  repetitions_ = options.get("repetitions").as<Index>();
+}
+
+std::pair<double, double> CalcStatistics(const std::vector<double>& v) {
+  double mean = std::reduce(v.begin(), v.end()) / v.size();
```

Comment:
```suggestion
  double mean = std::reduce(v.begin(), v.end()) / ((double)v.size());
```
probably the current approach gives compiler issues